simple/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 description: Helm chart with simple deployment/service template
 name: simple
-version: 0.18.1
+version: 0.18.2
```

Comment:
please increase minor version to 0.19.0, the last number is patch version

Reply:
Thanks for reminder. Updated.